File: Indie Python Programming Course/step03/3.01/3.1.03.py
'''

Вводятся два целых числа, каждое в отдельной строке.

Ваша задача вывести наибольшее из данных чисел.

Примечание: используйте только условный оператор, функцией max пользоваться нельзя

Sample Input 1:

8
11

Sample Output 1:

11

Sample Input 2:

50
21

Sample Output 2:

50

'''

a, b = int(input()), int(input())
if a>b:
    print(a)
else:
    print(b)

File: JavaScript. A3 Tasks/3 JavaScript для начинающих/3.01/3.1.03.js
/*

Вписать в указанное место код, который будет присваивать переменной "х" значение суммы переменных "a" и "b".

Указанные ниже Sample Input (образец входных данных) и Sample Output (образец выходных данных) - это пример. 
Они показывают, что при значениях переменных "a" и "b" как в Sample Input, например 2 и 8, ваш результат должен быть такой, как указано в Sample Output. 
Например, если Sample Input равен 2 и 8, то Sample Output должен быть 10, т.е. сумма первой и второй переменных.

Sample Input:

9 6

Sample Output:

15

*/

function testSum(a, b) {
    let x;
    x = a + b;
    return x;
}

let [a, b] = prompt().split(' ').map(Number)
console.log(testSum(a, b))
